daml-script/test/src/test-utils/com/daml/lf/engine/script/test/AbstractFuncIT.scala

```diff
@@ -4,15 +4,18 @@
 package com.daml.lf.engine.script
 package test
 
+import com.daml.bazeltools.BazelRunfiles
```

Comment:
Could you create a separate test for upgrade ? 

We will need to split dev and non dev test, you could copy from `MultiParticipantIT

Reply:
done, thanks